packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -254,6 +255,27 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     return res;
   }
 
+  @override
+  Future< List<AuthUserAttribute>> fetchUserAttributes({AuthUserAttributeRequest request}) async {
```

Comment:
extra space in `Future< List<AuthUserAttribute>>`

Reply:
Thanks.